packages/playground/blueprints/src/lib/resources.ts

```diff
@@ -164,7 +164,7 @@ export class VFSResource extends Resource {
 
 	/** @inheritDoc */
 	get name() {
-		return this.resource.path;
+		return this.resource.path.split('/').pop() || '';
```

Comment:
The resource returned a full path as a file name and that threw off installAsset